Backend/main.go

```diff
@@ -9,6 +10,8 @@ import (
 func main() {
 	http.HandleFunc("/edit", service.EditEndpoint)
 	http.HandleFunc("/preview", service.PreviewHTTPHandler)
+	http.HandleFunc("/getallusers", auth.GetAllUsers)
```

Comment:
is this implementation of auth a prototype implementation or an actual implementation? if its an actual implementation it should not be in the hacky code I wrote for the backend prototype, u could probably make a new branch of main and start from there instead?